libs/dh/imbalance-prices/shell/src/lib/drawer/dh-drawer.component.ts

```diff
@@ -109,50 +109,48 @@ import { format } from 'date-fns';
     WattButtonComponent,
     WattSpinnerComponent,
     VaterFlexComponent,
+
     DhStatusBadgeComponent,
     DhEmDashFallbackPipe,
     DhTableDayViewComponent,
-    DhFeatureFlagDirective,
   ],
 })
 export class DhImbalancePricesDrawerComponent {
   private readonly toastService = inject(WattToastService);
   private readonly httpClient = inject(ImbalancePricesHttp);
   private readonly apollo = inject(Apollo);
 
-  @ViewChild(WattDrawerComponent)
-  drawer: WattDrawerComponent | undefined;
+  private readonly year = computed(() => Number(format(this.imbalancePrice()!.name, 'yyyy')));
+  private readonly month = computed(() => Number(format(this.imbalancePrice()!.name, 'M')));
 
   imbalancePrice = input<DhImbalancePrice>();
+
   imbalancePricesForMonth = signal<DhImbalancePricesForMonth[]>([]);
   isLoading = signal(false);
+  lastUpdated = computed(() => {
+    const [firstDay] = this.imbalancePricesForMonth();
```

Comment:
Praise: Nice way of getting the first value in the array

Reply:
The joy of working with modern JavaScript :)